core/storage/statistics/gae_models.py

```diff
@@ -98,8 +97,6 @@ class AnswerSubmittedEventLogEntryModel(base_models.BaseModel):
     client_time_spent_in_secs = ndb.FloatProperty()
     # Whether the submitted answer received useful feedback
     is_feedback_useful = ndb.BooleanProperty(indexed=True)
-    # Whether the submitted answer is the correct answer.
-    is_answer_correct = ndb.BooleanProperty(indexed=True)
```

Comment:
Er -- you can't just remove fields, for GAE models. That will cause an error with existing data. To simulate, try recording some events on develop, then switch to this branch. (I can't remember whether this only happens in production or whether we've encountered it in development too, but we've definitely encountered it.)

Instead, just mark the fields as deprecated. See e.g. ExplorationModel -- we've deprecated some fields in the past.

Reply:
@seanlip. AnswerSubmittedEventLogEntryModel is not an existing GAE model. We created it in milestone 1. The existing AnswerSubmittedEvent handler only recorded answers, didn't create an event model.

Reply:
Ah ok. Then we can safely remove it, since there's no production data for this. Thanks for clarifying!